api/src/test/java/compilation/IsNotCompatibleWithJava8.java

```diff
@@ -0,0 +1,17 @@
+package compilation;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class IsNotCompatibleWithJava8 {
+/*
+ * This code segment is compatible with Java 11 and not compatible with java 8. It will abort the compilation if the Java context is version 8
+ */
+    @Test
+    public void test() {
+        String value = "This code is compiled in Java 11";
+        System.out.println(value.isBlank());
+    }
```

Comment:
I don't understand this. Maybe this might be useful - https://stackoverflow.com/questions/14572287/testing-different-java-versions-in-junit#answer-14573109

Reply:
This class uses a method (isBlank()) which is available only in JAVA 11. In java 8 the compilation sends an incompatibility message on the method.
The stackoverfow proposal is also very good. You just have to locate the right class to implement it. Do you have any suggestion ?

Reply:
I don't think we need a test for that. I would suggest to remove it.

Reply:
The question is in which scenario someone can run tests using JDK 8, when we are requesting 11 in pom.xml file. It is of course possible, but completely out of VIVO installation/building procedure, should we take care in that case? Anyway, if some test is needed, it shouldn't just print some message, we need assertion (probably in this case assertFalse(value.isBlank()), but not sure what will happen in JDK 1.8 case).